.travis.yml

```diff
@@ -3,6 +3,7 @@ language: node_js
 node_js:
   - 12
 script:
+  - yarn format
```

Comment:
Please revert this. Formatting code before `test:format` defeats the whole purpose of the format check :)

Reply:
done, reverted ;)